src/parsing/css-translation-parsing.js

```diff
@@ -19,7 +19,9 @@
       if (coords.length != 3) {
         return null;
       }
-      return [new CSSTranslation(coords[0], coords[1], coords[2]), remaining];
+      var result = [new CSSTranslation(coords[0], coords[1], coords[2]), remaining];
+      result[0]._inputType = '3';
```

Comment:
I really don't like this thing where we're modifying the private attributes of the result outside the css-translation.js file - this can get hard to reason about very quickly. What if instead we had internal-only functions `internal.buildCSSTranslation(x, y, z)` and `internal.buildCSSTranslationX(x)` etc, in css-translation.js, that attach the correct cssText method to the resulting object? I'm imagining something like this

```javascript

function CSSTranslation(x, y, z) {...}
...
function buildCSSTranslationX(x) {
  var result = new CSSTranslation(x, zeroLength);
  Object.defineProperty(result, 'cssText', {
    get: function() { return cssTextX(this); },
    set: function(newText) {}
  });
  return result;
}

internal.buildCSSTranslationX = buildCSSTranslationX;
```

This also gets rid of the magic '1', '2', 'x' etc, which I also think is not so great.